salt/utils/boto3mod.py

```diff
@@ -103,7 +99,7 @@ def _get_profile(service, region, key, keyid, profile):
     if not keyid and _option(service + ".keyid"):
         keyid = _option(service + ".keyid")
 
-    label = "boto_{}:".format(service)
+    label = "boto3_{}:".format(service)
```

Comment:
Will this change anything for users using any boto modules? Will they all of a sudden see changes when re-running states when they didnt intend there to be changes, etc?

Reply:
This label is used to generate the key that the connection is cached under in the `__context__`, this change is done to avoid clobbering `botomod` entries.  This can potentially be split out into a separate bugfix PR.

Reply:
Okay, so we should be good then. This would only impact current connections in `__context__`, which someone would not have a current connection if they upgrade salt, so I think this is fine. thanks for the explanation :)